.github/workflows/tests-ipsec-upgrade.yaml

```diff
@@ -20,9 +20,6 @@ on:
   push:
```

Comment:
Drive-by comment: should we also update this in e2e tests https://github.com/cilium/cilium/blob/main/.github/workflows/tests-e2e-upgrade.yaml#L478-L484?
I know it's unrelated to ipsec-upgrade, just wanted to check we're not missing pieces 🤔

Reply:
We can do that when we start covering upgrades for IPv6 underlay configs. Now that v1.18 branched out, maybe we can.

Anyway, it doesn't fail right now because we don't run the downgrade for IPv6 underlay configs.